specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/examples/CosmosDBLocationGet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2021-03-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "westus"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.DocumentDB/locations/westus",
+        "type": "Microsoft.DocumentDB/locations",
+        "name": "West US",
```

Comment:
>West US [](start = 17, length = 7)

The name must match the one in the id, which is 'westus'

Reply:
Updated